assets/quick-order-list.js

```diff
@@ -1,307 +1,281 @@
-if (!customElements.get('quick-order-list-remove-button')) {
-  customElements.define(
-    'quick-order-list-remove-button',
-    class QuickOrderListRemoveButton extends BulkAdd {
-      constructor() {
-        super();
-        this.addEventListener('click', (event) => {
-          event.preventDefault();
-          this.startQueue(this.dataset.index, 0);
-        });
-      }
-    }
-  );
-}
```

Comment:
Replaced this with a simple button so that events would be processed in the same QuickOrderList queue as the other quantity updates.